.github/workflows/pull-request-comments.yml

```diff
@@ -164,16 +164,16 @@ jobs:
 
             github.rest.issues.createComment( commentInfo );
 
-  # Leaves a comment on a pull request when no Trac ticket is included in the pull request description.
+  # Leaves a comment on a pull request when no Trac ticket is included in the pull request description and delete the comment if the Trac ticket has been updated.
   trac-ticket-check:
-    name: Comment on a pull request when no Trac ticket is included
+    name: Comment on a pull request when no Trac ticket is included and delete the comment when Trac Ticket has been updated
```

Comment:
Let's adjust this a bit. It's possible that more tasks are added to this in the future. So let's be more general for the `name` and move the details into the description.

Reply:
Fixed